src/client/lazy-app/worker-bridge/index.ts

```diff
@@ -58,7 +58,7 @@ for (const methodName of methodNames) {
           signal.removeEventListener('abort', onAbort);
 
           // Start a timer to clear up the worker.
-          this._workerTimeout = setTimeout(() => {
+          this._workerTimeout = window.setTimeout(() => {
```

Comment:
Was this necessary for some reason?

Reply:
TypeScript was complaining. For some reason it could not distinguish between node's setTimeout and browser's setTimeout. This also happened after I fooled around with different zip libraries. Will also remove